api/ImageCapture.json

```diff
@@ -0,0 +1,366 @@
+          "support": {
+            "webview_android": {
+              "version_added": "61"
+            },
+            "chrome": {
+              "version_added": "61"
+            },
+            "chrome_android": {
+              "version_added": "61"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "46"
+            },
+            "opera_android": {
+              "version_added": "46"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "grabFrame": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageCapture/grabFrame",
+          "support": {
+            "webview_android": {
+              "version_added": "59"
+            },
+            "chrome": {
+              "version_added": "59"
+            },
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "46"
+            },
+            "opera_android": {
+              "version_added": "46"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": "7.0"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "takePhoto": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageCapture/takePhoto",
+          "support": {
+            "webview_android": {
+              "version_added": "59",
+              "notes": "<code>photoSettings</code> argument added in version 60"
```

Comment:
I'll admit I'm a little conflicted about how to handle this `photoSettings` argument but after looking at the spec this is what I settled on. Rather than a note on each browser for the addition of the argument, I think it might be better to restructure this as two entries for each browser that supported it:

- an entry with a `version_added` value (in this case, 60)
- an entry with `partial_implementation` set to `true`, both a `version_added` (59) and `version_removed` (60), and a note saying the `photoSetting` argument wasn't supported